scripts/plots/mesoscale/exevs_mesoscale_sref_grid2obs_plots.sh

```diff
@@ -136,12 +195,19 @@ for fcst_valid_hour in $fcst_valid_hours ; do
 	 #*********************
          > run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.sh  
 
+	#*******************************************************************************************************************
+	#  Check if this sub-job has been completed in the previous run for restart
+	 if [ ! -e $restart/run_${stats}.${score_type}.${lead}.${VAR}.${FCST_LEVEL_value}.${line_type}.${fcst_valid_hour}.completed ] ; then
+	#******************************************************************************************************************* 
+   	 
 	if [ $VAR = UGRD ] || [ $VAR = VGRD ] || [ $VAR = HGT ] || [ $VAR = TMP ] ; then
 	   verif_type=upper_air
         else	   
            verif_type=conus_sfc
         fi
 
+      	
+
```

Comment:
@ShelleyMelchior-NOAA Just pointing out some extra spaces to be removed before this PR is merged. Thanks!